### DISCUSSION TITLE
Welcome to MoveCertificate Discussions!

### DISCUSSION THREAD

**Discussion by ys1231** [ys1231/MoveCertificate#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xiaobaiweinuli** [comment]
web 界面如何打开？
https://mui.kernelsu.org/index.html显示错误

**Comment by ys1231** [comment]
这个是kernelsu的错误 不是我的，之前我也遇到过，重新刷内核安装最新版kernelsu就解决了

**Reply by xiaobaiweinuli** [reply]
感谢
